pages/docs/concepts/_meta.json

```
@@ -0,0 +1,5 @@
+{
+  "anchor-system": "Anchor System",
+  "distributed-key-gen": "Distributed Key Generation (DKG)",
+  "tss": "Threshhold Signature Scheme (TSS)"
```

```suggestion
  "tss-governance": "Threshhold Signature Scheme (TSS)"
```

Next.js references <file: name> so it should match the file name you want to be displayed. For example, tss-governance.mdx is the file you want to display so it should be referenced accordingly in the _meta.json otherwise it will render both files where `Threshhold Signature Scheme (TSS)` is empty.

---

CC @thomivy